design/drive-status.md

```diff
@@ -0,0 +1,141 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# hardware-status
+
+## Status
+
+provisional
+
+## Table of Contents
+
+   * [Title](#title)
+      * [Status](#status)
+      * [Table of Contents](#table-of-contents)
+      * [Summary](#summary)
+      * [Motivation](#motivation)
+         * [Goals](#goals)
+         * [Non-Goals](#non-goals)
+      * [Proposal](#proposal)
+         * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints-optional)
+         * [Risks and Mitigations](#risks-and-mitigations)
+      * [Design Details](#design-details)
+         * [Work Items](#work-items)
+         * [Dependencies](#dependencies)
+         * [Test Plan](#test-plan)
+         * [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+         * [Version Skew Strategy](#version-skew-strategy)
+      * [References](#references)
+
+## Summary
+
+This document explains how information about hardware components (drives, power
+supplies, fans, etc.) including health data, will be retrieved, collected and
+monitored to provide constant up-to-date status on hardware devices present in
+a server.
```

Comment:
For reviewers of this document who were not involved in the conference call where we talked about a design, it would be useful to have a brief description in this section. It doesn't need to go into all of the details, but should explain that we intend to run a service on each host in the cluster, that the service will collect data and post it to ironic-inspector, and that the baremetal operator will consume it as part of the regular polling for status that it is already performing.

Reply:
I will add a short paragraph explaining the base idea.